src/components/ListNestedComponent.tsx

```diff
@@ -21,8 +21,14 @@ export class ListNestedComponent extends React.Component<any, any> {
     }
 
     render(): any {
-        if (this.props.field.conditionalField && this.props.field.conditionalValue !== this.props.model[this.props.field.conditionalField]) {
-            return <div />
+        if (this.props.field.conditionalField) {
+            if (Array.isArray(this.props.field.conditionalValue)) {
+                if (!_.includes(this.props.field.conditionalValue, this.props.model[this.props.field.conditionalField])) {
+                    return <div />
+                }
+            } else if (this.props.field.conditionalValue !== this.props.model[this.props.field.conditionalField]) {
+                return <div />
+            }
```

Comment:
In Table, It will be displayed only when the conditional Value matches the conditional Field

Supported for both (array type conditional value) and single conditional value